### DISCUSSION TITLE
"Nuxt-like" routing

### DISCUSSION THREAD

**Discussion by ldiebold** [quasarframework/quasar#12382] [Quasar App Extensions]
# Quasar "Nuxt-like" Routing is here!

Ever wanted auto-routing in Quasar? Kinda like Nuxt?
Well now it's possible with Quasar [auto-routing](https://github.com/ldiebold/quasar-app-extension-auto-routing)

To get started:
```sh
quasar ext add auto-routing
```

And use the generated routes in your `routes.js` file:
```js
import autoRouting from './auto-routing' // 💥

const routes = [
  ...autoRouting, // 💥

  // Always leave this as last one,
  // but you can also remove it
  {
    path: '/:catchAll(.*)*',
    component: () => import('pages/Error404.vue')
  }
]

export default routes
```

[Find out more on the github repo!](https://github.com/ldiebold/quasar-app-extension-auto-routing)

This is still new, so feedback/PRs most welcome!

**Comment by goldelehr** [comment]
Hi, 

I noticed a rather annoying bug. When using this extension,  "quasar build" won't exit after successfully building the Quasar app (it just prints the build tips and never quits). 

Apart from that your extension works really great and helped me a lot keeping me sane while working with many routes.
